src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ChangeEvent, FC, ReactNode } from 'react';
+import { Checkbox as ChakraCheckbox } from '@chakra-ui/react';
+
+interface Props {
+  name: string;
+  value?: string;
```

Comment:
shouldn't `value` and `onChange` be mandatory props for components that are controlled from the outside?

Reply:
I think is fine: https://github.com/smg-automotive/components-pkg/pull/108#discussion_r915837317